site/content/api/Tenant-API.md

```diff
@@ -315,24 +400,28 @@ The table below contains the properties which are used to configure a *Hono prot
 
 Protocol Adapters SHOULD use the configuration properties set for a tenant when interacting with devices of that tenant, e.g. in order to make authorization decisions or to limit message rates per tenant etc.
 
-Additionally to the specified properties the JSON object MAY contain an arbitrary number of members with arbitrary names which can be of a scalar or a complex type. 
+The JSON object MAY contain an arbitrary number of additional members with arbitrary names of either scalar or complex type.
 
 ### Examples
 
-Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT`.
+Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT`. Note that the payload contains some *custom* properties at both the tenant as well as the adapter configuration level.
```

Comment:
Better name the *custom* properties explicitly?
Would make it easier for readers that are completely new to the tenant API to identify them (and is not much effort).